src/cljs/rems/application.cljs

```diff
@@ -416,7 +417,9 @@
     (rf/dispatch [::set-field id (.. event -target -value)])))
 
 (defn- id-to-name [id]
-  (str "field" id))
+  (if-not id
+    (str "id_missing")
+    (str "field" id)))
```

Comment:
ID should be a required attribute, so instead of returning a dummy value (that will likely conflict with others), it's safer to throw an exception.